docs/writing-stories/parameters.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Parameters'
+---
+
+**Parameters** are a set of static, named metadata about a story, typically used to control the behaviour of Storybook features and addons.
+
+For example, let’s customize the backgrounds addon via a parameter. We’ll use `parameters.backgrounds` to define which backgrounds appear in the backgrounds toolbar when a story is selected.
+
+### Story parameters
+
+We can set a parameter for a single story with the `parameters` key on a CSF export:
+
+```js
+export const Primary = Story.bind({});
+Primary.args = …
+Primary.parameters = { backgrounds: … };
```

Comment:
We should probably put legitimate values for the backgrounds addon in here.

Reply:
let me know if you're ok with the new update on this snippet. I've updated it recently

Reply:
LGTM!